hyperstart.go

```diff
@@ -139,16 +143,53 @@ func (h *hyper) buildHyperContainerProcess(cmd Cmd) (*hyperJson.Process, error)
 	return process, nil
 }
 
-func bindMountContainerRootfs(pod Pod, container ContainerConfig) error {
-	rootfsDest := filepath.Join(defaultSharedDir, pod.id, container.ID)
+func (h *hyper) buildPauseBinName(pauseName, podID, cID string) string {
+	return fmt.Sprintf("%s-%s-%s", pauseName, podID, cID)
+}
+
+func (h *hyper) bindMountContainerRootfs(container ContainerConfig) error {
+	rootfsDest := filepath.Join(defaultSharedDir, h.pod.id, container.ID)
+
+	err := bindMount(container.RootFs, rootfsDest)
+	if err != nil {
+		return err
+	}
 
-	return bindMount(container.RootFs, rootfsDest)
+	linkPauseBinPath := filepath.Join(rootfsDest, rootfsDir, h.buildPauseBinName(pauseBinName, h.pod.id, container.ID))
+
+	_, err = os.Stat(linkPauseBinPath)
+	if err == nil {
+		return nil
+	}
+
+	return bindMount(h.config.PauseBinPath, linkPauseBinPath)
```

Comment:
I honestly don't understand why we need to do this second bind mount. Please have a look at what I have in mind with the pause container here:

https://github.com/containers/virtcontainers/issues/83